WalletWasabi.Fluent/Helpers/MathUtils.cs

```diff
@@ -9,4 +9,22 @@ public static int Round(int n, int precision)
 		var rounded = roundedFraction * precision;
 		return (int)rounded;
 	}
-}
\ No newline at end of file
+
+	public static decimal RoundToSignificantFigures(this decimal num, int n)
```

Comment:
Can you move this whole file to `WalletWasabi\Helpers\` and write some basic unit tests for the added method?